### DISCUSSION TITLE
Profile Option: Hide All Contribution Activity

### DISCUSSION THREAD

**Discussion by criticalsoft** [community/community#7689] [Other Feature Feedback, Questions, & Ideas]
This is definitely a feature that many developers need.
(For privacy)

**Comment by radar-blip** [comment]
There is a checkbox for this in settings, down near the bottom.

![hideit](https://user-images.githubusercontent.com/39196767/178419610-096e3928-8bcf-4d82-8719-3033116bac0d.png)

**Reply by criticalsoft** [reply]
Thank You. It's Working.

**Reply by radar-blip** [reply]
You're welcome. 👍🏼
